### DISCUSSION TITLE
Keep pages State in NavigationView

### DISCUSSION THREAD

**Discussion by AxelCope** [bdlukaa/fluent_ui#760] [Q&A] [answered]
Hello, I started Fluent a while ago, and I noticed that the states of the pages were not kept when switching between them, how to fix it? THANKS

**Comment by bdlukaa** [comment]
You can use `NavigationView.paneBodyBuilder` with an `IndexedStack`. But, with the most recent versions, the pages should be kept alive automatically

**Comment by AxelCope** [comment]
Thank you for the answer. I've currently upgraded to Fluent 4.5.0, but my problem still persists, states are still not preserved when I switch between pages. I was then obliged to use the Tabs which keep the states for the moment, but it still annoys me that the pane does not do it. I may be doing it wrong. Check this project out with a minimal essay to show you my method of using the pane and if I'm doing any wrong(https://github.com/AxelCope/fluenttest). THANKS.

**Reply by bdlukaa** [reply, marked as answer]
You can make use of `AutomaticKeepAliveClientMixin`
